gr-fec/lib/ccsds_encoder_impl.cc

```diff
@@ -28,9 +28,7 @@
 #include <gnuradio/fec/generic_encoder.h>
 #include <cstdio>
 
-extern "C" {
```

Comment:
um, wut? viterbi.h pretty much starts out with a `namespace` statement. Thanks for spotting this one!

Reply:
I added the namespace to convert it to real C++...